### DISCUSSION TITLE
More current OS versions?

### DISCUSSION THREAD

**Discussion by compi2k** [mastodon/mastodon#31059] [General]
As of now Mastodon only supports Ubuntu 20.04 and Debian 11 officially. Both OS will go EOL "rather soon", the latter has only roughly two years left (I know its still a long time, but still).

When is it planned to support newer Distros officially?

**Comment by mjankowski** [comment]
I believe this is resolved in latest release and we're on current stable ... but if not and this is still relevant, can you add pointers to exact version support you'd want and how/where it's not supported?

**Comment by compi2k** [comment]
Yes Thank You!
